workbench

```diff
@@ -963,6 +982,36 @@ def update_media() -> None:
                 {'value': status}
             ]
         }
+    
+    def is_revisions_enabled(config: dict, media_type: str) -> bool:
```

Comment:
I don't think the type hint `-> bool` is compatible with Workbench's current minimum required version of Python, 3.7. We need to update to 3.8 soon though. Can you remove this for now please?